Add compatibility with >=pillow-6.0.0 

Changed VERSION to PILLOW_VERSION,
as it got removed with >=pillow-6.0.0.

Taken from https://github.com/OpenTTD/nml/pull/29
--- a/nml/version_info.py
+++ b/nml/version_info.py
@@ -71,11 +71,11 @@
     #PIL
     try:
         from PIL import Image
-        versions["PIL"] = Image.VERSION
+        versions["PIL"] = Image.PILLOW_VERSION
     except ImportError:
         try:
             import Image
-            versions["PIL"] = Image.VERSION
+            versions["PIL"] = Image.PILLOW_VERSION
         except ImportError:
             versions["PIL"] = "Not found!"
 
